client/post-editor/editor-categories-tags/test/accordion.jsx

```diff
@@ -13,150 +13,171 @@ import useMockery from 'test/helpers/use-mockery';
 import useFakeDom from 'test/helpers/use-fake-dom';
 
 describe( 'EditorCategoriesTagsAccordion', function() {
-	let shallow, common, categoryStore, tagStore, i18n, CategoriesTagsAccordion, accordion;
+	let mount, common, tagStore, i18n, accordion, EditorCategoriesTagsAccordion;
 
 	useMockery();
 	useFakeDom();
 
 	before( () => {
-		mockery.registerMock( 'post-editor/editor-categories', EmptyComponent );
+		mockery.registerMock( 'post-editor/editor-term-selector', EmptyComponent );
 		mockery.registerMock( 'post-editor/editor-tags', EmptyComponent );
 		mockery.registerMock( 'components/info-popover', EmptyComponent );
+		mockery.registerMock( 'react-virtualized/VirtualScroll', EmptyComponent );
 
-		shallow = require( 'enzyme' ).shallow;
+		mount = require( 'enzyme' ).mount;
```

Comment:
So the prior version of the tests was creating an instance of the class and testing the out put of `getSubtitle()`.  I tried to use `shallow` here but it appears the the `<Accordion />` was not fully being rendered as I could not find the node with the title: `accordion.find( '.accordion__subtitle' )`.

Granted this was my first time using `enzyme` so I could very well not know the proper way to use it here.